docs/src/components/Layout.js

```diff
@@ -68,8 +68,8 @@ const MarkdownContainer = (props) => {
   const { relativePagePath, is404, pageTitle, location } = props;
 
   return (
-    <Row data-test="Docs-inner--container" className="justify-content-center">
-      <Column className="px-11 py-8 min-vh-100 inner-left-container" size={9}>
+    <Row className="justify-content-center">
+      <Column data-test="Docs-inner--container" className="px-11 py-8 min-vh-100 inner-left-container" size={9}>
```

Comment:
The same `data-test` ID is already used above. Please write appropriate ID to differentiate the both